app/src/main/java/com/deframe/artapp/helper/MuseumListAdapter.kt

```diff
@@ -54,12 +70,23 @@ class MuseumListAdapter(val museumList: ArrayList<Museum>) : RecyclerView.Adapte
         fun bindItems(data: Museum) {
             val txtName = itemView.findViewById<TextView>(R.id.list_item_museumName)
             val txtAddress = itemView.findViewById<TextView>(R.id.list_item_museumAddress)
+            val img = itemView.findViewById<ImageView>(R.id.list_item_museumImage)
```

Comment:
Please check for ``itemView != null`` else may give you ``nullPointerException``.